src/python/foglamp/core/service_registry/monitor.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+"""FogLAMP Monitor module"""
+
+import asyncio
+import aiohttp
+import logging
+import os
+import json
+
+from foglamp import logger
+from foglamp import configuration_manager
+from foglamp.core.service_registry.instance import Service
+
+__author__ = "Ashwin Gopalakrishnan"
+__copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+# Module attributes
+_CONNECTION_STRING = "dbname='foglamp' user='foglamp'"
+
+try:
+  snap_user_common = os.environ['SNAP_USER_COMMON']
+  unix_socket_dir = "{}/tmp/".format(snap_user_common)
+  _CONNECTION_STRING = _CONNECTION_STRING + " host='" + unix_socket_dir + "'"
+except KeyError:
+  pass
+
```

Comment:
do we need this here? Any db operation etc. ? and actually storage layer basic operations are in place now (with some basic local manual work)